src/pages/Article/ArticlePage.jsx

```diff
@@ -33,37 +19,37 @@ function ArticlePage({ article }) {
           justifyContent: 'center',
           alignItems: 'center',
           height: 'calc(100% - 60px)',
-          overflowY: 'auto',
-          backgroundColor: '#FF0000',
+          overflowY: 'auto'
         }}
       >
         <Box
           sx={{
             height: '80%',
-            width: '40%',
+            width: '60%',
             display: 'flex',
+            flexDirection: 'column',
             justifyContent: 'center',
             alignItems: 'center',
             backgroundColor: '#D9D9D9',
             padding: '20px',
-            overflowY: 'auto',
           }}
         >
-          <fieldset>
-            <legend>{articleDisplay.title} </legend>
-            <div style={{ padding: '5px' }}>
-              {articleDisplay.content}
-              <p>
-                <a
-                  href={articleDisplay.link}
-                  target='_blank'
-                  rel='noopener noreferrer'
-                >
-                  Visit Website
-                </a>
-              </p>
-            </div>
-          </fieldset>
+          <div
+            style={{
+              border: '1px solid black',
+              padding: '5px',
+              overflow: 'hidden',
+              overflowY: 'auto',
+            }}
+          > Article content here
```

Comment:
Article content here is displayed all the time and does not make sense

Reply:
removed